src/components/MultiLink.vue

```diff
@@ -241,23 +249,29 @@ export default Vue.extend({
     },
 
     dragNode(node: Node, event: MouseEvent) {
-      event.preventDefault();
+      if (!(this.$refs.svg instanceof Element)) { throw new Error('SVG is not of type Element'); }
+
+      event.stopPropagation();
 
       const moveFn = (evt: Event) => {
+        // Check we have a mouse event
+        if (!(evt instanceof MouseEvent)) { throw new Error('event is not MouseEvent'); }
+
         // eslint-disable-next-line no-param-reassign
-        node.x = (evt as MouseEvent).clientX - this.controlsWidth - (this.calculateNodeSize(node) / 2);
+        node.x = evt.x - this.controlsWidth - (this.calculateNodeSize(node) / 2);
         // eslint-disable-next-line no-param-reassign
-        node.y = (evt as MouseEvent).clientY - (this.calculateNodeSize(node) / 2);
+        node.y = evt.y - (this.calculateNodeSize(node) / 2);
```

Comment:
Did you mean to replace `.client[X|Y]` with `.[x|y]`?

Reply:
Yes, just for consistency across functions. I should have made an explicit commit for that